source/iterators.tex

```diff
@@ -6520,6 +6523,28 @@
 \pnum \returns \tcode{N}.
 \end{itemdescr}
 
+\indexlibrary{\idxcode{ssize(C\& c)}}%
+\begin{itemdecl}
+template <class C>
+constexpr auto ssize(const C& c)
+    -> common_type_t<ptrdiff_t, make_signed_t<decltype(c.size())>>;
```

Comment:
Indent only 2 spaces (as it appears in the declaration in the synopsis).

Reply:
Fixed.